charts/nfdiv-case-api/values.preview.template.yaml

```diff
@@ -8,6 +8,7 @@ java:
   environment:
     CASE_DATA_STORE_BASEURL: http://{{ .Release.Name }}-ccd-data-store-api
     CITIZEN_UPDATE_CASE_STATE_ENABLED: true
+    ENABLE_CDAM: false
```

Comment:
I think this is not part of your PR ?

Reply:
Its just to disable cdam for preview instance
I thought it would be useful to keep at least until the QA can be done, because they will not be able to see docs on the case off the PR build

Reply:
this will be true for all the preview testing though so worth creating a PR to disable it and let the QA's know to enable/disable it while testing ?

Reply:
OK - will remove from here